### PR TITLE
[IA-3270][IA-3271] Reapply igvjs update and memory leak fix

### PR DESCRIPTION
**Pull request by jordansafer**
After rolling back IA-3270 and IA-3271, we found that these changes were not the source of the problem. This PR is to reapply these changes.

Initial PRs:
https://github.com/DataBiosphere/terra-ui/pull/2909
https://github.com/DataBiosphere/terra-ui/pull/2911

Changes:
- Fix a memory leak in loading the IGV viewer
- Upgrade IGVjs to the latest version

### CODE REVIEW

**Review by nawatts — APPROVED** [review]

File: src/components/IGVBrowser.js
Comment (nawatts) [inline src/components/IGVBrowser.js]: Since `igvLibrary.current = igv` is set after an async import, it's probably worth checking if `igvLibrary.current` is set before calling `removeAllBrowsers` in case someone quickly closes the IGV browser before igv.js loads.

Comment (jordansafer) [reply src/components/IGVBrowser.js]: Ooh, that wouldn't be great. Updated-

Comment (kyuksel) [reply src/components/IGVBrowser.js]: I wonder if this would be a good use case for the `useGetter()` hook.

Comment (nawatts) [reply src/components/IGVBrowser.js]: `useGetter` stores a state variable in a ref so that a function can capture the ref and, through that ref, have access to the latest value of the state variable. Without the ref, the function would capture the value of the state variable at the time when the function was created. Since `igvLibrary` is a ref, and thus stable across renders, there's no need to wrap it in `useGetter`.

Comment (kyuksel) [reply src/components/IGVBrowser.js]: Ah right; that makes sense. Thanks for the clarification!

**Comment by kyuksel** [comment]
@jordansafer did you want to post around for an approval from a relevant team?

**Comment by jordansafer** [comment]
> @jordansafer did you want to post around for an approval from a relevant team?

@kyuksel, I shared in my team channel last week, but it looks like it slipped by :) . I'll find another pair of eyes-

**Review by davidangb — APPROVED** [review]
lgtm, though recent happenings have shown I don't know how to test all the use cases for IGV

**Pull request merged**
